packages/vaadin-lumo-styles/src/components/map.css

```diff
@@ -0,0 +1,442 @@
+}
+
+.ol-overviewmap {
+  grid-area: overview-map;
+  align-self: end;
+  width: max-content;
+}
+
+.ol-overviewmap button span:empty::before {
+  content: var(--vaadin-map-icon-overview-map-collapse, '\25BE');
+}
+
+.ol-overviewmap.ol-collapsed button span:empty::before {
+  content: var(--vaadin-map-icon-overview-map-expand, '\25B4');
+}
+
+.ol-overviewmap-map {
+  height: 10em;
+  width: 10em;
+  margin: var(--lumo-space-xs);
+  border: 0;
+  border-radius: var(--lumo-border-radius-s);
+}
+
+.ol-overviewmap.ol-collapsed .ol-overviewmap-map,
+.ol-overviewmap.ol-uncollapsible button {
+  display: none;
+}
+
+.ol-overviewmap-box {
+  border: 1px dashed rgba(0, 0, 0, 0.5);
+  filter: drop-shadow(0 0 1px #fff) drop-shadow(0 0 1px #fff);
+  will-change: filter;
+  cursor: move;
+}
+
+.ol-overviewmap:not(.ol-collapsed),
+.ol-overviewmap:not(.ol-collapsed):hover {
+  background-color: var(--lumo-base-color);
+  box-shadow: var(--lumo-box-shadow-s);
+  transition: 0.15s box-shadow;
+}
+
+.ol-overviewmap:not(.ol-uncollapsible) .ol-overviewmap-map {
+  margin-bottom: 0;
+}
+
+.ol-zoomslider {
+  grid-area: zoom-slider;
+  height: 8em;
+}
+
+.ol-zoomslider button {
+  position: relative;
+  display: block;
+  border-radius: inherit;
+  height: var(--lumo-space-m);
+}
+
+.ol-zoomslider:hover button {
+  box-shadow: var(--lumo-box-shadow-s);
+}
+
+.ol-zoomslider:not(.ol-uncollapsible):hover {
+  box-shadow: none;
+  overflow: visible;
+}
+
+.ol-zoom-extent {
+  grid-area: zoom-extent;
+  align-self: end;
+}
+
+.ol-control,
+.ol-scale-bar,
+.ol-scale-line {
+  margin: var(--lumo-space-xs);
+}
+
+.ol-control {
+  margin: 0.25em;
+  border-radius: var(--lumo-border-radius-m);
+  transition:
+    0.15s box-shadow,
+    0.15s background-color;
+  -webkit-backdrop-filter: blur(8px);
+}
+
+.ol-control:hover {
+  background-color: var(--lumo-base-color);
+}
+
+.ol-control:not(.ol-uncollapsible):hover {
+  box-shadow: var(--lumo-box-shadow-s);
+  background-color: var(--lumo-shade-20pct);
+}
+
+.ol-control button {
+  -webkit-appearance: none;
+  border: 0;
+  margin: 0;
+  padding: 0;
+  background: #fff;
+  font: inherit;
+  color: inherit;
+  width: var(--lumo-size-s);
+  height: var(--lumo-size-s);
+  border-radius: inherit;
+  font-family: 'lumo-icons';
+  font-size: var(--lumo-icon-size-s);
+  font-weight: 400;
+}
+
+.ol-control button::-moz-focus-inner {
+  border: none;
+  padding: 0;
+}
+
+.ol-control button,
+.ol-attribution:not(.ol-uncollapsible) ul {
+  transition: 0.15s opacity;
+  background-color: var(--lumo-base-color);
+  color: var(--lumo-body-text-color);
+  opacity: 0.65;
+}
+
+.ol-control:hover button,
+.ol-control button:focus,
+.ol-attribution:hover ul {
+  opacity: 1;
+}
+
+.ol-control button:hover {
+  color: var(--lumo-primary-text-color);
+}
+
+.ol-control button:active {
+  background: var(--lumo-base-color) linear-gradient(var(--lumo-contrast-5pct), var(--lumo-contrast-5pct));
+}
+
+@supports not selector(:focus-visible) {
```

Comment:
BTW we can probably remove this as all browsers now support `:focus-visible`.

Reply:
Good catch, done in https://github.com/vaadin/web-components/pull/9503